baseplate/lib/experiments/__init__.py

```diff
@@ -81,51 +97,59 @@ def __init__(
         config_watcher: FileWatcher,
```

Comment:
do we want to make this optional now so people can start migrating to 2.0 style before it's required?

Reply:
the point is i wonder if people just initiate the object without argument 
Experiment(filewatcher, span, name)?